tests/test_geoprocessing.py

```diff
@@ -285,8 +294,314 @@ def test_reproject_vector(self):
         vector = ogr.Open(target_vector_path)
         layer = vector.GetLayer()
         result_reference = layer.GetSpatialRef()
+
+        layer = None
+        vector = None
+
+        self.assertTrue(
+            osr.SpatialReference(result_reference.ExportToWkt()).IsSame(
+                osr.SpatialReference(target_reference.ExportToWkt())))
+
+    def test_reproject_vector_partial_fields(self):
+        """PGP.geoprocessing: reproject vector with partial field copy."""
+        import pygeoprocessing
+        # Create polygon shapefile to reproject
+        base_srs = osr.SpatialReference()
+        base_srs.ImportFromEPSG(3157) # NAD83(CSRS) / UTM zone 10N
+        extents = [443723.1273278, 4956276.905980, 443993.1273278, 4956546.905980]
+
+        polygon_a = shapely.geometry.Polygon([
+            (extents[0], extents[3]), (extents[0], extents[1]),
+            (extents[2], extents[1]), (extents[2], extents[3]),
+            (extents[0], extents[3])])

```

Comment:
It looks like this is constructing a box using the `extents` noted above.  What you have here totally works, but I just wanted to  mention that `shapely` has a convenience function [`shapely.geometry.box`](https://shapely.readthedocs.io/en/latest/manual.html#shapely.geometry.box) that will do the same thing in a slightly more compact notation.

Reply:
Great suggestion done in commit: [f2c337b](https://github.com/natcap/pygeoprocessing/pull/31/commits/f2c337bce218e8c7f7d922edf0eb32f4d962f6df)